<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="screen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>961</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn0">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn9">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_btn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>150</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>BusyCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="plus_btn">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="div_btn">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minus_btn">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>40</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="umn_btn">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>150</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>*</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>40</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="dot_btn">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>.</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="percent_btn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>BusyCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="del_btn">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>40</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>⌫</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="enter_btn">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>150</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>↵</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="sqrt_btn">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>260</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>√</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Калькулятор</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
